shared/src/main/scala/fetch.scala

```diff
@@ -205,9 +162,9 @@ object `package` {
      * results. It implies concurrent execution of fetches.
      */
     def join[A, B](fl: Fetch[A], fr: Fetch[B]): Fetch[(A, B)] =
-      Free.liftF(Join(fl, fr))
+      Free.liftF[FetchOp, (A, B)](Join(fl, fr))
 
-    class FetchRunner[M[_]] {
+    private[fetch] class FetchRunner[M[_]](private val dummy: Boolean = true) extends AnyVal {
```

Comment:
I'm curious, why we need this `dummy` value?

Reply:
That is a trick to make `FetchRunner` a value class that is used in Cats as well.

Reply:
Great, thanks for the responses